src/platform_impl/windows/event_loop.rs

```diff
@@ -2051,10 +2055,6 @@ unsafe extern "system" fn thread_event_target_callback<T: 'static>(
 ) -> LRESULT {
   let subclass_input = Box::from_raw(subclass_input_ptr as *mut ThreadMsgTargetSubclassInput<T>);
 
-  if msg != WM_PAINT {
-    RedrawWindow(window, ptr::null(), HRGN::default(), RDW_INTERNALPAINT);
-  }
```

Comment:
what's the reason for removing this?

Reply:
This signal is more used for sending clear events. In its `WM_PAINT` arm, it checks if there are events still need to handle.
Other windows should also have their own `WM_PAINT` arm to handle drawing event.
I'll open an issue on winit to ask about if this is safe.

Reply:
sounds good